src/nvim/vim.h

```diff
@@ -338,14 +338,9 @@ enum {
  * functions of these names. The declarations would break if the defines had
  * been seen at that stage.  But it must be before globals.h, where error_ga
  * is declared. */
-#if !defined(FEAT_GUI_W32) && !defined(FEAT_GUI_X11) \
-  && !defined(FEAT_GUI_GTK) && !defined(FEAT_GUI_MAC)
 # define mch_errmsg(str)        fprintf(stderr, "%s", (str))
 # define display_errors()       fflush(stderr)
 # define mch_msg(str)           printf("%s", (str))
-#else
-# define USE_MCH_ERRMSG
```

Comment:
```USE_MCH_ERRMSG```  is referenced in files ```src/nvim/globals.h``` and ```src/nvim/message.c```

And because of ```scripts/stripdecls.py``` I thought an user can add or remove this const at compile time by adding an argument, so i didn't deleted it. 

I didn't understand the purpose of scripts/stripdecls.py and so I wouldn't remove this const until anybody say "Yes you can fully delete it"